public/v1/apps/drone.json

```diff
@@ -0,0 +1,102 @@
+{
+    "captainVersion": "1",
+    "documentation": "Taken from https://hub.docker.com/r/drone/drone/",
```

Comment:
This is just a link to their Docker Hub page, it doesn't include any docs. Usually we use this "documentation" field to point to the URL where the original Docker Compose file was taken. This will help us to address potential issues in the future as we can reach out to the original provider and ask for their advice.

If you manually created this Docker Compose file, that's fine too. Just post a link to the docs where this composition is explained (agent and server).

Let me know what you think.

Reply:
agreed